hpy/devel/include/common/hpytype.h

```diff
@@ -31,6 +31,18 @@ typedef struct {
     HPyDef **defines;   /* points to an array of 'HPyDef *' */
 } HPyType_Spec;
 
+typedef enum {
+    HPyType_SpecParam_Base = 1,
+    HPyType_SpecParam_Bases = 2,
+    //HPyType_SpecParam_Metaclass = 3,
+    //HPyType_SpecParam_Module = 4,
+} HPyType_SpecParam_Kind;
+
+typedef struct {
+    HPyType_SpecParam_Kind kind;
+    HPy object;
```

Comment:
what about using `void *` instead of `HPy`?
I can't think of any reason to use something different than `HPy` right now, but why close the door for the future?

Reply:
And use ``HPy_AsVoidP()``?  That seems strange to require this API, which is advanced and maybe not really supported on all implementations, just for that.

Reply:
Right, I didn't think that we can't cast `HPy` to `void *` directly. Then ok, I am fine with leaving it as it is.